bond/commands/upgrade.py

```diff
@@ -120,7 +124,8 @@ def run(self, args):
         branch = get_branch_string(args.branch, target)
         print(f"Selected Branch: \t{branch}")
         print(f"Current Version: \t{current_ver}")
-        version_obj = get_latest_version(target, branch)
+        rollback = int(args.rollback) or 0
```

Comment:
This should be handled in the argument spec, you can add a type and default: `"type": int, "default": 0`